cybersecuritytools/csls/hec_index_checker/hec_index_checker.py

```diff
@@ -45,10 +50,10 @@ def hec_index_checker(accounts: str, token: str, ssm: str) -> bool:
     except ClientError:
         print("[!] Unable to build Splunk certificate bundle")
         return False
-
+    print(cert_bundle)
```

Comment:
will this cause any secrets to be in logs?

Reply:
The `cert_bundle` in this context is public X509 certificates. They are not secret, but the print shouldn't be there either.